doc/quick-start-user-guide.md

```diff
@@ -149,15 +157,24 @@ Send the volume create request to create a 2x2 distributed-replicate volume:
 $ curl -X POST http://192.168.56.101:24007/v1/volumes --data @volcreate.json -H 'Content-Type: application/json'
 ```
 
-### Start the volume
+Send the volume create request using glustercli:
+
+    $ ./glustercli volume create testvol <uuid1>:/export/brick1/data <uuid2>:/export/brick2/data --force
```

Comment:
Please remove `--force`

Reply:
This example does not match the curl equivalent example (bricks).

Reply:
made changes assuming that the number of bricks was the issue here?
also added the replica flag as in the curl example.